content/docs/learning-guide/tutorials/rendering/get-started-materialtypes-and-shaders.md

```diff
@@ -37,119 +35,134 @@ The `.material` file is created using **Material Editor** later in the tutorial.
 {{< /note >}}
 
 ## 2. Author AZSL Shader Code (.azsl)
+
 First, you are going to write AZSL shader code in the `MyUnlitColor.azsl` file. The AZSL shader contains the following components:
+
 - #include directives
 - Shader Resource Groups (SRGs)
 - Vertex and fragment shader inputs and outputs
 - Vertex and fragment shader programs
 
 #### Includes
-At the very top of the file are a set of `#include` directives. A shader program can use shared Shader Resource Groups (SRGs) and reusable AZSL code, which are contained in .srgi and .azsli files. 
+
+At the very top of the file are a set of `#include` directives. A shader program can use shared Shader Resource Groups (SRGs) and reusable AZSL code, which are contained in .srgi and .azsli files.
 
 For this tutorial, you need the following #include directives:
 
-* `viewsrg.srgi`: Defines a shared view SRG.
-* `DefaultObjectSrg.azsli`: Defines an object SRG.
-* `ForwardPassOutput.azsli`: Defines a ForwardPassOutput struct. 
-* `SrgSemantics.azsl`: Defines SRG semantics. Anytime you want to define an SRG, the SRG must inherit from an SRG semantic. 
+- `viewsrg.srgi`: Defines a shared view SRG.
+- `DefaultObjectSrg.azsli`: Defines an object SRG.
+- `ForwardPassOutput.azsli`: Defines a ForwardPassOutput struct.
+- `SrgSemantics.azsl`: Defines SRG semantics. Anytime you want to define an SRG, the SRG must inherit from an SRG semantic.
 
 To use #include directives, add the following code in the `MyUnlitColor.azsl` file:
 
-```glsl
+```cpp
 #include <viewsrg.srgi>
 #include <Atom/Features/PBR/DefaultObjectSrg.azsli>
 #include <Atom/Features/PBR/ForwardPassOutput.azsli>
 #include <Atom/Features/SrgSemantics.azsli>
```

Comment:
Asset processor failed to build the example without these updates. The full example code below needs to be updated as well.

```suggestion
#pragma once

#include <viewsrg.srgi>
#include <Atom/Features/PBR/DefaultObjectSrg.azsli>
#include <Atom/Features/Pipeline/Forward/ForwardPassOutput.azsli>
#include <Atom/Features/SrgSemantics.azsli>
```